adapters/inmobi/inmobi.go

```diff
@@ -118,18 +118,17 @@ func preprocess(imp *openrtb2.Imp) error {
 	return nil
 }
 
-func getMediaTypeForImp(impId string, imps []openrtb2.Imp) openrtb_ext.BidType {
-	mediaType := openrtb_ext.BidTypeBanner
-	for _, imp := range imps {
-		if imp.ID == impId {
-			if imp.Video != nil {
-				mediaType = openrtb_ext.BidTypeVideo
-			}
-			if imp.Native != nil {
-				mediaType = openrtb_ext.BidTypeNative
-			}
-			break
-		}
+func getMediaTypeForImp(bid openrtb2.Bid) openrtb_ext.BidType {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner
+	case openrtb2.MarkupVideo:
+		return openrtb_ext.BidTypeVideo
+	case openrtb2.MarkupAudio:
+		return openrtb_ext.BidTypeAudio
+	case openrtb2.MarkupNative:
+		return openrtb_ext.BidTypeNative
+	default:
+		return openrtb_ext.BidTypeBanner
```

Comment:
It appears you were defaulting to banner before adding `mtype` support but now that you have it, is your server supposed to always set it? If so, you might want to do what most other adapters do which is to return an error if `mtype` does not equal one of your supported formats.
```
default:
	return "", fmt.Errorf("unrecognized bid type in response from inmobi for bid %s", bid.ImpID)
```
and then in the calling code:
```
mediaType, err := getMediaTypeForImp(sb.Bid[i].ImpID, internalRequest.Imp)
if err != nil {
	return nil, []error{err}
}
```
Thoughts?

Reply:
Sounds good. I've made the recommended changes.